Deploy-CAM.ps1

```diff
@@ -2349,6 +2414,38 @@ function Deploy-CAM() {
 				},
 				"secretName": "cloudAccessRegistrationCode"
 			}
+		},
```

Comment:
Seems to be some tabs versus spaces thing going on in this section? I think @cptera changed the whole file recently to 4-spaces indents.